tests/common/server_fixture.rs

```diff
@@ -402,6 +416,37 @@ impl TestServer {
                     .expect("set ID failed");
 
                 println!("Set writer_id to {:?}", id);
+
+                // if server ID was set, we can also wait until DBs are loaded
+                let check_dbs_loaded = async {
+                    let mut interval = tokio::time::interval(Duration::from_millis(500));
+
+                    loop {
+                        if management_client
+                            .get_server_status()
+                            .await
+                            .unwrap()
+                            .initialized
+                        {
+                            return;
+                        }
+                        interval.tick().await;
+                    }
```

Comment:
noob question: any subtle rust gotcha for why isn't this written like:

```suggestion
                    while !management_client
                            .get_server_status()
                            .await
                            .unwrap()
                            .initialized {
                        interval.tick().await;
                    }
```

e.g. the .await cannot be in the loop condition or something like that?